iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/SubstationXml.java

```diff
@@ -84,12 +79,14 @@ protected Substation readRootElementAttributes(SubstationAdder adder, NetworkXml
                 .map(c -> context.getAnonymizer().deanonymizeCountry(Country.valueOf(c)))
                 .orElse(null);
         String tso = context.getAnonymizer().deanonymizeString(context.getReader().getAttributeValue(null, "tso"));
-        String geographicalTags = context.getReader().getAttributeValue(null, "geographicalTags");
-        if (geographicalTags != null) {
-            adder.setGeographicalTags(Arrays.stream(geographicalTags.split(","))
-                    .map(tag -> context.getAnonymizer().deanonymizeString(tag))
-                    .toArray(size -> new String[size]));
-        }
+        IidmXmlUtil.runUntilMaximumVersion(IidmXmlVersion.V_1_2, context, () -> {
+            String geographicalTags = context.getReader().getAttributeValue(null, "geographicalTags");
+            if (geographicalTags != null) {
+                adder.setGeographicalTags(Arrays.stream(geographicalTags.split(","))
```

Comment:
Don't use `setGeographicalTags` method (to be deprecated), store it in a String and store it in properties after creating the substation.